guide/payments/units-and-symbols.md

```diff
@@ -0,0 +1,146 @@
+amount of different notation standards around the world, it focuses on bitcoin.
+
+Illustration sources
+
+https://www.figma.com/file/VB3GQdAnhl8yta44DY3PSV/Bitcoin-Wallet-UI-Kit?node-id=2318%3A61126
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/payments/units-and-symbols/units-and-symbols.png"
+   retina = "/assets/images/guide/payments/units-and-symbols/units-and-symbols@2x.png"
+   mobile = "/assets/images/guide/payments/units-and-symbols/units-and-symbols-mobile.png"
+   mobileRetina = "/assets/images/guide/payments/units-and-symbols/units-and-symbols-mobile@2x.png"
+   alt-text = "Bitcoin value with annotations for parts of the number display."
+   caption = "Aspects to consider when formatting bitcoin amounts."
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+# Units and symbols
+
+Bitcoin, bits, sats. The format and presentation of bitcoin values are probably amongst the most challenging for new users. While there are no official standards for how they should be used, a good way to achieve converging user experiences is to formalize what is widely used today.
+
+
+#### Current adoption
+
+The most common way bitcoin is expressed is as BTC (bitcoin) or sat (satoshi), with 1 bitcoin being 100 million satoshi. The unicode symbol ₿, formalized in June 2017, is also used to represent BTC (bitcoin), but typeface support is still limited. While not as common, other denominations of BTC such as mBTC (millibitcoins), μBTC (bits) are sometimes used, as well as msat (millisatoshi) in the Lightning network. The chart below illustrates how each unit relates to the bitcoin unit.
+
+| Unit         | Symbol       | Bitcoin value     |
+| ------------ | ------------ | ----------------- |
+| bitcoin      | BTC or ₿     | 1.00 000 000      |
+| millibit     | mBTC         | 0.00 100 000      |
+| bit          | μBTC         | 0.00 000 100      |
+| satoshi      | sat          | 0.000 000 01      |
+| millisatoshi | msat         | 0.000 000 000 01  |
+
+For more information, see the Bitcoin Wiki:
+
+- [Units](https://en.bitcoin.it/wiki/Units)
+- [Satoshi](https://en.bitcoin.it/wiki/Satoshi_(unit))
+
+#### Recommended interaction
+
+When displaying bitcoin values, the default unit for on-chain wallets should be bitcoin with 8 decimal places, and satosho for Lightning wallets. Due to the challenging nature of scanning a monetary value with more than 2 decimal places, the user should be given the option to choose their preferred format across the application (for example, in the settings section) as well as contextually, whenever the value is primarily displayed.
+
+Product teams can choose an approach based on their audience and targeted use case. Lightning wallets for daily spending may be better served by defaulting to satoshi due to the low amounts involved, while the opposite can be true for savings-focused applications.
```

Comment:
```suggestion
Product teams can choose an approach based on their audience and targeted use case. Lightning wallets for daily spending may be better served by defaulting to satoshi due to the low amounts involved, while bitcoin, or bits can be used for savings-focused applications.
```

Opposite meaning? Here I think we need to be a bit more specific.

Reply:
No need to mention 'bits' as a recommendation here I think (very rarely used, not something I'd want to encourage). Otherwise suggestion makes sense as it is clearer.

Reply:
```suggestion
Product teams can choose an approach based on their audience and targeted use case. Lightning wallets for daily spending may be better served by defaulting to satoshi due to the low amounts involved, while bitcoin can be used for savings-focused applications.
```